static/app/components/events/interfaces/crashContent/exception/rawContent.tsx

```diff
@@ -1,20 +1,16 @@
 import {Component, Fragment} from 'react';
```

Comment:
This file is responsible for actually wrangling the stacktrace into the stacktrace string that's displayed when "Raw stacktrace" is selected. I have moved two methods, `getContent` and `getAppleCrashreportEndpoint` to the `utils.tsx` file to make for better reusability.